ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/WriteService.scala

```diff
@@ -100,5 +101,5 @@ trait WriteService
       transactionMeta: TransactionMeta,
       transaction: SubmittedTransaction,
       estimatedInterpretationCost: Long,
-  ): CompletionStage[SubmissionResult]
+  )(implicit telemetryContext: TelemetryContext): CompletionStage[SubmissionResult]
```

Comment:
I am all for this change but I'd like some verification that it's OK to change this interface (and if so, how often, and do we need backwards compatibility?). @gerolf-da, could you chime in?

Reply:
The general approach we have been using so far is as follows:
* Add a new method with the changed signature.
* Mark the original signature as deprecated and remove it in the next release (i.e., in this case in 1.14).

Reply:
Done. I had a problem distinguishing the methods. Any better idea how to solve this @miklos-da @SamirTalwar-DA?

Reply:
Let's discuss this offline.

Reply:
@SamirTalwar-DA: FYI, @miklos-da is following up offline with the relevant stakeholders. Most integrations use a kvutils interface, and we can talk directly to the users of this particular interface and get their opinion on it.

Reply:
@gerolf-da @hubert-da @SamirTalwar-DA We concluded we do not need to deprecate any `WriteService` methods but add an implicit `TelemetryContext` parameter where needed.